engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -349,7 +355,7 @@ private void preRenderUpdate(RenderingStage renderingStage) {
      * to provide statistics regarding the ongoing rendering and its individual steps (i.e. rendering shadows,
      * reflections, 2D filters...).
      *
-     * @param renderingStage "MONO" for standard rendering and "LEFT_EYE" or "RIGHT_EYE" for stereoscopic displays.
+     * @param renderingStage "MONO" for standard rendering and "leftEye" or "rightEye" for stereoscopic displays.
```

Comment:
To be more clear, I was swatting CheckStyle violations in openvrprovider and that was caught by a pattern match.